pkg/webui/components/dropdown/index.js

```diff
@@ -100,49 +100,58 @@ const DropdownItem = ({
   exact,
   showActive,
   tabIndex,
-  external,
   submenuItems,
+  external,
   ...rest
 }) => {
   const [expandedSubmenu, setExpandedSubmenu] = useState(false)
+  const [leaveDelayTimer, setLeaveDelayTimer] = useState(null)
   const iconElement = icon && <Icon className={style.icon} icon={icon} nudgeUp />
-  const activeClassName = classnames({
-    [style.active]: (!Boolean(action) && showActive) || active,
-  })
-  const cls = useCallback(
-    ({ isActive }) => classnames(style.button, { [activeClassName]: isActive }),
-    [activeClassName],
-  )
```

Comment:
There's no need to show any active items in the dropdown, so I've removed all logic that will visually mark items corresponding to an active route.